client/maisie/cli/users.py

```diff
@@ -42,21 +38,20 @@ def rm():
 @click.option(
     "-id", "--id", default=None, type=int, help="Returns user with a specified id"
 )
-def ls(id):
+@click.pass_context
+def ls(context, id):
     if id:
         users = Users().get(id)
     else:
         users = Users().get_all()
 
     if users:
-        table = SingleTable(
-            Transform().api_response_to_terminaltables(
-                users, include=DEFAULT_DISPLAY_ATTRIBUTES
-            )
+        title = "List of users"
+        include = DEFAULT_DISPLAY_ATTRIBUTES
+        display = Display(
+            context=context, response=users, attributes=include, title=title
```

Comment:
I think that instead of creating an additional variable called `include` we can now pass the `DEFAULT_DISPLAY_ATTRIBUTES` itself directly to the Display method, using the supported `attributes` parameter.